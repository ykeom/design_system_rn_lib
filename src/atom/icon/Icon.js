import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class Button extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>icon</Text>
      </View>
    );
  }
}

export default Button;
