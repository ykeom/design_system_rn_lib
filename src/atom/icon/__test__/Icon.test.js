import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Icon from '../Icon';

describe('Icon', () => {
  it('should render', () => {
    const component = shallow(<Icon />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
