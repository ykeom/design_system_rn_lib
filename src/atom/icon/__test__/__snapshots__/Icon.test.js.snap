// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    icon
  </Text>
</View>
`;
